homeassistant/components/vizio/config_flow.py

```diff
@@ -135,38 +168,36 @@ def __init__(self) -> None:
+                    if not errors:
+                        return await self._create_entry_if_unique(user_input)
+                # pylint: disable=no-member # https://github.com/PyCQA/pylint/issues/3167
+                elif self._must_show_form and self.context["source"] == SOURCE_IMPORT:
+                    # Import should always display the config form if CONF_ACCESS_TOKEN
+                    # wasn't included but is needed so that the user can choose to update
+                    # their configuration.yaml or to proceed with config flow pairing. We
+                    # will also provide contextual message to user explaining why
+                    _LOGGER.warning(
+                        "Couldn't complete configuration.yaml import: '%s' key is missing. To "
+                        "complete setup, '%s' can be obtained by going through pairing process "
+                        "via frontend Integrations menu; to avoid re-pairing your device in the "
+                        "future, once you have finished pairing, it is recommended to add "
+                        "obtained value to your config ",
+                        CONF_ACCESS_TOKEN,
+                        CONF_ACCESS_TOKEN,
+                    )
                     self._must_show_form = False
                 else:
-                    # Abort flow if existing entry with same unique ID matches new config entry.
-                    # Since name and host check have already passed, if an entry already exists,
-                    # It is likely a reconfigured device.
-                    unique_id = await VizioAsync.get_unique_id(
-                        user_input[CONF_HOST],
-                        user_input.get(CONF_ACCESS_TOKEN),
-                        user_input[CONF_DEVICE_CLASS],
-                        session=async_get_clientsession(self.hass, False),
-                    )
-
-                    if await self.async_set_unique_id(
-                        unique_id=unique_id, raise_on_progress=True
-                    ):
-                        return self.async_abort(
-                            reason="already_setup_with_diff_host_and_name"
-                        )
+                    self._data = copy.deepcopy(user_input)
+                    return await self.async_step_pair_tv()
 
-                    return self.async_create_entry(
-                        title=user_input[CONF_NAME], data=user_input
-                    )
-
-        # Use user_input params as default values for schema if user_input is non-empty, otherwise use default schema
         schema = self._user_schema or _get_config_schema()
 
+        # pylint: disable=no-member # https://github.com/PyCQA/pylint/issues/3167
+        if errors and self.context["source"] == SOURCE_IMPORT:
+            # Log an error message if import config flow fails since otherwise failure is silent
+            _LOGGER.error("configuration.yaml import failure: %s", errors.values())
```

Comment:
`values()` is a generator and cannot be stringified. Use `', '.join(errors.values())`